qemu/tests/multi_disk_random_hotplug.py

```diff
@@ -346,9 +349,50 @@ def verify_qtree_unsupported(params, info_qtree, info_block, qdev):
         return logging.warn("info qtree not supported. Can't verify qtree vs. "
                             "guest disks.")
 
+    def enable_driver_verifier(driver, timeout=300):
+        return utils_test.qemu.windrv_check_running_verifier(
+            session, vm, test, driver, timeout)
+
+    def _initial_win_drives():
+        size = params['stg_image_size']
+        disks = utils_disk.get_windows_disks_index(session, size)
+        if not utils_disk.update_windows_disk_attributes(session, disks):
+            test.fail("Failed to update windows disk attributes.")
+        for index, disk in enumerate(disks, 1):
+            if index == 24:
+                break
+            yield utils_disk.configure_empty_windows_disk(session, disk, size)[0]
+
+    def do_stress_iozone():
+        error_context.context("Do iozone stress after hotplug", logging.info)
+        try:
+            iozone = generate_instance(params, session, 'iozone')
+            iozone_cmd_option = params['iozone_cmd_option']
+            iozone_timeout = float(params['iozone_timeout'])
+            for letter in _initial_win_drives():
+                iozone.run(iozone_cmd_option.format(letter), iozone_timeout)
+        finally:
+            iozone.clean()
+
+    def do_stress_dd():
+        error_context.context("Do dd stress after hotplug", logging.info)
+        output = session.cmd_output(params.get("get_dev_cmd", "ls /dev/[svh]d*"))
+        system_dev = re.findall(r"/dev/[svh]d\w+(?=\d+)", output)[0]
+        for dev in re.split(r"\s+", output):
+            if not dev:
+                continue
+            if not re.findall(system_dev, dev):
+                dd_cmd = 'dd of={0} if=/dev/null bs=1M count=1000 oflag=direct '
+                dd_cmd + '&& dd of=/dev/zero if={0} bs=1M count=1000 oflag=direct'
+                session.cmd(dd_cmd.format(dev), timeout=1800)
+
     vm = env.get_vm(params['main_vm'])
     qdev = vm.devices
     session = vm.wait_for_login(timeout=int(params.get("login_timeout", 360)))
+    os_type = params['os_type']
+    if os_type == 'windows':
+        session = enable_driver_verifier(params['driver_name'])
```

Comment:
Variable 'driver_name' may not exist when testing types other than virtio_blk/virtio_scsi, like ide.